Gemfile.lock

```diff
@@ -42,7 +42,8 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/bgs-ext.git
-  revision: 4e216f8097fe93c4ca76d06f358c8d169afcd7ad
+  revision: ec28b2c269444fd42d9851a9c600c5eecd24552e
+  branch: API-15184-ClaimsApi-insertintenttofile-allow-optional-clmantSsn-attribute
```

Comment:
same as above

Reply:
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/9810/commits/a1dee3955147e8d42b26c8c651c479c5712c5566).